src/java/io/bazel/rulesscala/coverage/instrumenter/JacocoInstrumenter.java

```diff
@@ -26,28 +25,18 @@
 import org.jacoco.core.instr.Instrumenter;
 import org.jacoco.core.runtime.OfflineInstrumentationAccessGenerator;
 
-public final class JacocoInstrumenter implements Processor {
+public final class JacocoInstrumenter implements Worker.Interface {
 
-    public static void main(String[] args) throws Exception {
-        (new Worker()).run(args);
-    }
-
-    private static final class Worker extends GenericWorker {
-        public Worker() {
-            super(new JacocoInstrumenter());
-        }
+    public static void main(String args[]) throws Exception {
```

Comment:
nit: do we have preference between `String[] args` and `String args[]`? I have seen both in this PR. I personally prefer the former.

Reply:
I am a bad Java programmer so I mix the two. Can fix, though!

Reply:
Fixed!